src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -103,7 +103,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                         group = 'Docker'
                         description = "Tags Docker image with tag '${tagName}'"
                         workingDir dockerDir
-                        commandLine 'docker', 'tag', '--force=true', ext.name, computeName(ext.name, tagName)
+                        commandLine 'docker', 'tag', ext.name, computeName(ext.name, tagName)
```

Comment:
Cool. I'll rebase!

> On Apr 4, 2016, at 7:56 PM, Mark Elliot <notifications@github.com> wrote:
> 
> In src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy:
> 
> > @@ -103,7 +103,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
> >                          group = 'Docker'
> >                          description = "Tags Docker image with tag '${tagName}'"
> >                          workingDir dockerDir
> > -                        commandLine 'docker', 'tag', '--force=true', ext.name, computeName(ext.name, tagName)
> > +                        commandLine 'docker', 'tag', ext.name, computeName(ext.name, tagName)
> Yeah, I think we should probably merge this now...but it needs to be brought up to date/checks need to pass.
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>